OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -319,10 +319,9 @@ public void Nudge(Actor self, Actor nudger, bool force)
 				if (cellInfo != null)
 				{
 					self.CancelActivity();
-					var notifyBlocking = new CallFunc(() => self.NotifyBlocker(cellInfo.Cell));
-					var waitFor = new WaitFor(() => CanEnterCell(cellInfo.Cell));
-					var move = new Move(self, cellInfo.Cell);
-					self.QueueActivity(ActivityUtils.SequenceActivities(notifyBlocking, waitFor, move));
+					self.QueueActivity(new CallFunc(() => self.NotifyBlocker(cellInfo.Cell)));
+					self.QueueActivity(new WaitFor(() => CanEnterCell(cellInfo.Cell)));
```

Comment:
Out of scope for this PR, but for future reference: the two above look like they should become child activities of `Move` at some point, or even not be their own activities at all: we could do `self.NotifyBlocker` in `OnFirstRun` and make `Tick` `return false` early if `!CanEnterCell`.